### PR TITLE
Extract strings for the config family of commands.

### PR DESCRIPTION
**Pull request by brendandburns**
@fabianofranz @kubernetes/sig-cli-pr-reviews

### CODE REVIEW

File: translations/kubectl/default/LC_MESSAGES/k8s.po [hunk 1]
```diff
@@ -38,3 +38,68 @@ msgstr[0] ""
 msgstr[1] ""
 "watch is only supported on individual resources and resource collections - "
 "%d resources were found"
+
+# https://github.com/kubernetes/kubernetes/blob/masterpkg/kubectl/cmd/config/config.go#L39
```
Comment (sttts) [inline translations/kubectl/default/LC_MESSAGES/k8s.po]: the paths lack a slash, don't they?

Comment (brendandburns) [inline translations/kubectl/default/LC_MESSAGES/k8s.po]: done.

File: pkg/kubectl/cmd/config/create_context.go [hunk 2]
```diff
@@ -54,7 +55,7 @@ func NewCmdConfigSetContext(out io.Writer, configAccess clientcmd.ConfigAccess)
 
 	cmd := &cobra.Command{
 		Use:     fmt.Sprintf("set-context NAME [--%v=cluster_nickname] [--%v=user_nickname] [--%v=namespace]", clientcmd.FlagClusterName, clientcmd.FlagAuthInfoName, clientcmd.FlagNamespace),
-		Short:   "Sets a context entry in kubeconfig",
+		Short:   i18n.T("Sets a context entry in kubeconfig"),
```
Comment (sttts) [inline pkg/kubectl/cmd/config/create_context.go]: The other string like `"Context %q set."` below will be in a follow-up?

Comment (sttts) [inline pkg/kubectl/cmd/config/create_context.go]: The other strings like `"Context %q set."` below will be in a follow-up?

**Comment by sttts** [comment]
/lgtm

Note though that there are a number of other strings around.

File: pkg/kubectl/cmd/config/create_context.go (hunk 2 above)
Comment (brendandburns) [inline pkg/kubectl/cmd/config/create_context.go]: Yep, many, many strings to go :)

Comment (brendandburns) [inline pkg/kubectl/cmd/config/create_context.go]: yep, miles to go before I sleep...

**Comment by brendandburns** [comment]
@sttts Can I get a re-lgtm due to rebasing? thanks!

**Comment by sttts** [comment]
@brendandburns just re-add lgtm to escape rebase hell if necessary.

**Comment by brendandburns** [comment]
@k8s-bot unit test this

**Comment by brendandburns** [comment]
/approve

(self-lgtm due to rebase)

**Comment by fabianofranz** [comment]
@k8s-bot unit test this

**Pull request merged**